Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM golang:1.18-buster as builder
+
+ENV TZ=America/New_York
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+
+# Update linux image which contains golang-1.18
```

Comment:
Does this mean we would need to be on a linux machine to launch/use this docker?

Reply:
It's actually the exact opposite.We can essentially create a build from any base image that exists in the registry [hub.docker.com/](https://hub.docker.com/search?q=) I basically run a ubuntu image as my mac and do essentially most of my day to day in linux [ubuntu](https://hub.docker.com/_/ubuntu)